src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/info/switches/LogicalPortInfoEntry.java

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.switches;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@JsonNaming(value = SnakeCaseStrategy.class)
+public class LogicalPortInfoEntry implements Serializable {
+    private Integer logicalPortNumber;
```

Comment:
Is these fields (`logicalPortNumber`, `type`, `physicalPorts`) are equal to `expected` or to `actual` objectes? If so, why for we need them?

Reply:
these fields are equal to actual objects. Why?
1. I just used approach from rule/meter/group validation
2. actual and expected contains only not equals field. (if type for actual and expected will be same - this field will be missed in actual and expected)
3. It will be strange to have `expected` field and do not have `actual state`